packages/web/components/complex/pool/create/step1-set-ratios.tsx

```diff
@@ -0,0 +1,70 @@
+import Image from "next/image";
+import { FunctionComponent } from "react";
+import { observer } from "mobx-react-lite";
+import classNames from "classnames";
+import { TokenSelect } from "../../../control";
+import { InputBox } from "../../../input";
+import { StepProps } from "./types";
+import { StepBase } from "./step-base";
+
+export const Step1SetRatios: FunctionComponent<StepProps> = observer(
+  (props) => {
+    const { createPoolConfig: config } = props;
+
+    return (
+      <StepBase step={1} {...props}>
+        <div className="flex flex-col gap-2.5">
+          {config.assets.map(({ amountConfig, percentage }, index) => (
+            <div
+              key={amountConfig.currency.coinDenom}
+              className="h-24 flex px-7 items-center place-content-between border border-white-faint rounded-2xl"
+            >
+              <TokenSelect
+                selectedTokenDenom={amountConfig.currency.coinDenom}
+                tokens={config.sendableCurrencies}
+                onSelect={(token) => {
+                  const currency = config.remainingSelectableCurrencies.find(
+                    (currency) => currency.coinDenom === token.coinDenom
+                  );
+                  if (currency) {
+                    amountConfig.setCurrency(currency);
+                  }
+                }}
+              />
+              <div className="flex items-center gap-2.5">
+                <InputBox
```

Comment:
how about adding type props for input box? For example, it would be better for ux to be able to handle numbers only for `<input type="number" ... />`.